BrickController2/BrickController2/PlatformServices/GameController/GameControllerEventArgs.cs

```diff
@@ -6,15 +6,19 @@ namespace BrickController2.PlatformServices.GameController
 {
     public class GameControllerEventArgs : EventArgs
     {
-        public GameControllerEventArgs(GameControllerEventType eventType, string eventCode, float value)
+        public readonly string ControllerDeviceId;
```

Comment:
Please keep the style of coding as this class typycally publish properties, it's not so common here to make class variable public.

Reply:
You're right - I've fixed it.
https://github.com/vicocz/brickcontroller2/pull/75/commits/203009cc5734ed6de1522f1a1a4faa4f78d6b842#diff-116a1abd7b9d8d41698ff629281dcf4f34c955948f13b0c5e8a57cef2af1f5fc